code/modules/client/client_colour.dm

```diff
@@ -67,11 +67,13 @@
 	if(CC)
 		client.color = CC.colour
 
-
-
+#define GLASS_CLIENT_COLOUR_PRIORITY 0
+#define EYE_CLIENT_COLOUR_PRIORITY 1
+#define CURSED_HEARTH_CLIENT_COLOUR_PRIORITY 100 //it's an indicator you're dying, so it's very high priority
+#define QUIRK_CLIENT_COLOUR_PRIORITY INFINITY //we can't see colors anyway!
```

Comment:
Why are you making these defines?

Reply:
Quite frankly, I don't know what I'm doing anymore.